build/sbt-launch-lib.bash

```diff
@@ -38,12 +38,11 @@ dlog () {
 
 acquire_sbt_jar () {
```

Comment:
Where did you get this file, or you just manually modified it?

Reply:
https://github.com/delta-io/delta/pull/642/files#diff-3d92b4f5f048a351dd0984abcc67b6bc54fffb54aaf8f505d5f95ce800d36e11